README.md

```diff
@@ -71,4 +71,9 @@ Follow these steps to set up the project locally:
    bin/dev # use to enable tailwind and hot reload
    ```
 
-5. Visit `http://localhost:3000` in your web browser to see the application running.
+5. Precompile the assets:
+   ```
+   rake assets:precompile
```

Comment:
To fix `ActionView::Template::Error (The asset "tailwind.css" is not present in the asset pipeline.)`